lib/services/models.js

```diff
@@ -111,14 +112,14 @@ function get(model, renderModel, executeRender, uri, locals) { /* eslint max-par
       }
 
       if (ms > timeoutLimit * 0.5) {
-        let slowMsg = ''
-          + `slow get ${uri} ${ms}ms`
-          + `\n  db.get:  ${times.afterGet - times.start}ms`
-          + `\n  upgrade: ${times.afterUpgrade - times.afterGet}ms`
-          + `\n  render:  ${times.afterRender - times.afterUpgrade}ms`;
+        const dbDuration = times.afterGet - times.start,
+          upgradeDuration = times.afterUpgrade - times.afterGet,
+          renderDuration = times.afterRender - times.afterUpgrade;
+        let detailedRenderTimes = '';
```

Comment:
This should be `[]` if it's an array of values.

Reply:
I think detailedRenderTimes should also be turned into an array of objects rather than a string.

Just like how you're translating the error message itself to an object - detailedRenderTimes should also be an array of objects rather than a formatted string

Reply:
and actuall instead of an array it should be an object of key/value pairs.  e.g.

```js
// from this
detailedRenderTimes = locals.amphoraRenderTimes
  .map(({ data, label, ms }) => {
    let msg = `  ${label}: ${ms}ms`;

    if (!_.isEmpty(data)) {
      msg += '\n' + formatTimingData(data);
    }

    return msg;
  })
  .join('\n');


// to this
detailedRenderTimes = locals.amphoraRenderTimes.reduce(
  (res, { label, ms }) => {
    res[label] = ms;
    return res;
  },
  {}
)
```

Reply:
ah sorry - just realized there may be scarnios where a loop causes the same label to be added multiple times, making the labels non-unique.  So an array should be used.  So you shouldn't have to do any processing on amphoraRenderTimes, you should be able to pass it as-is